internal/generator/vector/apiaudit/helpers_test.go

```diff
@@ -0,0 +1,113 @@
+package apiaudit
+
+import (
+	"bytes"
+	"encoding/json"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/cluster-logging-operator/test"
+	"k8s.io/apimachinery/pkg/runtime"
+	. "k8s.io/apiserver/pkg/apis/audit/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const vectorImage = "quay.io/openshift-logging/vector:5.8.0"
```

Comment:
There is a function in utils to get component images by name

Reply:
fixed